scenarios/iam_privesc_by_rollback/cheat_sheet_raynor.md

```diff
@@ -1,9 +1,9 @@
-`aws configure --profile raynor`
+`aws configure --profile Raynor`
```

Comment:
This is minor but looks like in other cases we are keeping the profile name lowercase. Let's update the lines below to match this instead.

Reply:
Actually it looks like this was already fixed in master, so yeah this would need changed back to be lowercase.